vg/vg.go

```diff
@@ -5,7 +5,7 @@
 // Package vg defines an interface for drawing 2D vector graphics.
 // This package is designed with the hope that many different
 // vector graphics back-ends can conform to the interface.
-package vg // import "gonum.org/v1/plot/vg"
+package vg
```

Comment:
please leave this import comment.
it makes sure people import "vg" via the vanity import path `"gonum.org/v1/plot/vg"`.